modules/apps/portal-vulcan/portal-vulcan-test/src/testIntegration/java/com/liferay/portal/vulcan/custom/field/test/CustomFieldsUtilTest.java

```diff
@@ -2155,6 +2261,508 @@ public void testToMapExpectedClassAndValueInvalidValues() {
 				null, null));
 	}
 
+	@Test
+	public void testToMapWithNonexistentCustomFieldAndLazyReferencingDisabled()
```

Comment:
the tests we did for the lazy enabled / disabled are just unique having in the same test both the conditions. But I don't know your standards 😄

Reply:
(the reason why I created only one)

Reply:
I also prefer different tests (for me it's more clear) but as you know there is another step to have it merged in master 😄